ts/features/bonus/bpd/saga/index.ts

```diff
@@ -37,15 +43,14 @@ export function* watchBonusBpdSaga(bpdBearerToken: string): SagaIterator {
   // load citizen details
   yield takeLatest(
     bpdLoadActivationStatus.request,
-    getCitizen,
-    bpdBackendClient.find
+    bpdTechnicalIban ? getCitizenV2 : getCitizen,
+    bpdBackendClient.findV2
   );
-
   // enroll citizen to the bpd
   yield takeLatest(
     bpdEnrollUserToProgram.request,
-    putEnrollCitizen,
-    bpdBackendClient.enrollCitizenIO
+    bpdTechnicalIban ? putEnrollCitizenV2 : putEnrollCitizen,
+    bpdBackendClient.enrollCitizenV2IO
```

Comment:
```suggestion
    bpdTechnicalIban
      ? bpdBackendClient.enrollCitizenV2IO
      : bpdBackendClient.enrollCitizenIO
```
The same as the previous comment.

Reply:
Modified in [a6d8119](https://github.com/pagopa/io-app/pull/2969/commits/a6d81191d57f6222c2b125346eff0afca5ec8829)